specifyweb/frontend/js_src/lib/components/FormEditor/fetchAllViews.ts

```diff
@@ -78,7 +82,14 @@ const augmentDatabaseView = (
       ? userInformation.availableCollections.find(
```

Comment:
githuv doesn't let me leave comments on lines that weren't modified for some weird reason, but regarding lines 77 and 78:
since ViewDefinition now includes collectionId, you don't need `& { readonly collectionId: number }` in here anymore

Reply:
and could you also do a quick check to see if we have `& { readonly collectionId: number }` anywhere else where it doesn't need to be anymore?